### PR TITLE
Mentor Requests API: Improve nesting and add solution uuid

### PR DESCRIPTION
**Pull request by iHiD**
https://forum.exercism.org/t/another-api-feature-request/8017

### CODE REVIEW

**Comment by glennj** [comment]
Your branch name does indeed make me happy :)

**Comment by iHiD** [comment]
@glennj Making you happy matters to me :)

File: app/javascript/components/mentoring/queue/useMentoringQueue.tsx [hunk 1]
```diff
@@ -8,12 +8,10 @@ import type { MentoredTrack, MentoredTrackExercise } from '@/components/types'
...
-  trackTitle: string
-  trackIconUrl: string
-  exerciseTitle: string
-  exerciseIconUrl: string
-  studentHandle: string
-  studentAvatarUrl: string
+  track: { title: string }
+  exercise: { title: string; iconUrl: string }
+  student: { handle: string; avatarUrl: string }
+  solution: { uuid: string }
```
Comment (ErikSchierboom) [inline app/javascript/components/mentoring/queue/useMentoringQueue.tsx:14]: Nice!

**Review by ErikSchierboom — APPROVED** [review]
I like it!

**Pull request merged**